React/CoreModules/RCTAccessibilityManager.mm

```diff
@@ -76,6 +76,13 @@ - (instancetype)init
                                                  name:UIAccessibilityReduceMotionStatusDidChangeNotification
                                                object:nil];
 
+      if (@available(iOS 14.0, *)) {
+          [[NSNotificationCenter defaultCenter] addObserver:self
+                                                   selector:@selector(prefersCrossFadeTransitionsStatusDidChange:)
+                                                       name:UIAccessibilityPrefersCrossFadeTransitionsStatusDidChangeNotification
+                                                     object:nil];
+      }
```

Comment:
This is not working as expected, `UIAccessibilityPrefersCrossFadeTransitionsStatusDidChangeNotification` never gets triggered. Best option might be just removing it for now and reading `UIAccessibilityPrefersCrossFadeTransitions()` directly

I've opened a question in Apple's Developer Forums reporting this issue https://developer.apple.com/forums/thread/699257